erpnext/stock/doctype/warehouse/warehouse.js

```diff
@@ -37,7 +37,8 @@ frappe.ui.form.on("Warehouse", {
 				function() { convert_to_group_or_ledger(frm); }, 'fa fa-retweet', 'btn-default')
 		}
 		
-		frm.toggle_enable(['is_group', 'company'], false);
+		frm.toggle_enable(['is_group'], false);
+		frm.toggle_enable(['company'], true);
```

Comment:
```suggestion
		frm.toggle_enable(['is_group'], false);
```

Try this out, this should work.